lsp-mode.el

```diff
@@ -1188,6 +1188,19 @@ Deprecated. Use `lsp-repeatable-vector' instead. "
   "Display lsp error message with FORMAT with ARGS."
   (message "%s :: %s" (propertize "LSP" 'face 'error) (apply #'format format args)))
 
+(defun lsp--fatal-error (format &rest args)
```

Comment:
seems like this is redundant: there is lsp--error.

Reply:
Ignore... I missed the fact that it is using error not message. But still, is this function used in this PR? Are we going to replace all error calls with that?

Reply:
`lsp--error` logs a `message`, which doesn't cause evaluation to abort.

Reply:
This function is used by this PR, which is why I added it. However, replacing all error calls might be a bit much. From looking at the code, the `(error (lsp-error ...))` idiom seems common, however it doesn't solve the text-properties issue.